inactive_maintainers/__init__.py

```diff
@@ -35,25 +32,20 @@ def main() -> None:
     print(gh.get_rate_limit(), file=stderr)
     org = gh.get_organization("nixos")
     nixpkgs = org.get_repo("nixpkgs")
+
     committers = org.get_team_by_slug("nixpkgs-committers").get_members()
     sorted_committers = sorted(list(committers), key=lambda c: c.login.lower())
 
-    blacklist = ["GrahamcOfBorg"]
```

Comment:
Introduced in https://github.com/nix-community/rfc55/commit/673c597fee20f4531e9558cb8a5e1db72e446946, but looks like this bot is not committer anyway.